From d76f35196b82ad6bb5c0a29920aac2765c63c9f3 Mon Sep 17 00:00:00 2001
From: Matt Selsky <matthew.selsky@twosigma.com>
Date: Sat, 3 Dec 2016 00:43:06 -0500
Subject: [PATCH 134/268] Massage shutil.get_terminal_size()'s os.terminal_size
 tuple into TermSize

os.terminal_size's members are columns, lines so we save to a new version and
then stuff them into TermSize

Fixes GitLab issue #185
---
 pylib/util.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pylib/util.py b/pylib/util.py
index ff648d1..04388eb 100644
--- a/pylib/util.py
+++ b/pylib/util.py
@@ -72,7 +72,8 @@ def termsize():
     if not os.isatty(1):
         size = (80, 24)
     try:
-        size = shutil.get_terminal_size((80, 24))
+        (w, h) = shutil.get_terminal_size((80, 24))
+        size = (w, h)
     except AttributeError:
         # OK, Python version < 3.3, cope
         import fcntl, termios, struct
-- 
2.7.4

